assets/js/main.js

```diff
@@ -474,6 +474,34 @@ function directory(jQuery) {
       {data: "videochat", title: "Video Chat", className: "dt-left", width: "18%"},
       {data: "pdf", title: "PDF", className: "dt-left", width: "8%"},
     ],
+
+    columnDefs: [
+        {
+            render(data, type, row) {
+                return `
+                    <button type="button" onclick="openJit('https://meet.jit.si/ohbm2020-${row.number}', ${row.number})">
+                        Open
```

Comment:
Could we replace `Open` with the text of the url like it was before with a simple link?  Some of them "non-standard" so visualizing that is of added information.

Reply:
done